src/main/java/me/senseiwells/essentialclient/clientscript/values/BiomeValue.java

```diff
@@ -0,0 +1,275 @@
+package me.senseiwells.essentialclient.clientscript.values;
+
+import me.senseiwells.arucas.api.ArucasClassExtension;
+import me.senseiwells.arucas.api.docs.ClassDoc;
+import me.senseiwells.arucas.api.docs.FunctionDoc;
+import me.senseiwells.arucas.throwables.CodeError;
+import me.senseiwells.arucas.utils.Arguments;
+import me.senseiwells.arucas.utils.ArucasFunctionMap;
+import me.senseiwells.arucas.utils.Context;
+import me.senseiwells.arucas.values.*;
+import me.senseiwells.arucas.values.functions.MemberFunction;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.registry.BuiltinRegistries;
+import net.minecraft.util.registry.RegistryKey;
+import net.minecraft.world.biome.Biome;
+import net.minecraft.world.biome.BiomeKeys;
+
+import java.util.Optional;
+
+import static me.senseiwells.arucas.utils.ValueTypes.NUMBER;
+import static me.senseiwells.essentialclient.clientscript.core.MinecraftAPI.*;
+
+public class BiomeValue extends GenericValue<Biome> {
+	public BiomeValue(Biome biome) {
+		super(biome);
+	}
+
+	@Override
+	public GenericValue<Biome> copy(Context context) {
+		return this;
+	}
+
+	@Override
+	public String getAsString(Context context) {
+		return "Biome{%s}".formatted(this.value.toString());

```

Comment:
Should just use string concatenation here, `.formatted` is really slow, and `toString` is a very common operation